// @breezingin/breezing-shared/dist/esm/schemas/admin-role.js
export var AdminPermission;
(function (AdminPermission) {
    // Workspace
    AdminPermission["AP_WorkspaceList"] = "listWorkspace";
    AdminPermission["AP_WorkspaceCreate"] = "createWorkspace";
    AdminPermission["AP_WorkspaceUpdate"] = "updateWorkspace";
    AdminPermission["AP_WorkspaceDelete"] = "deleteWorkspace";
    AdminPermission["AP_WorkspaceImport"] = "importWorkspace";
    AdminPermission["AP_WorkspaceEventList"] = "listWorkspaceEvent";
    AdminPermission["AP_WorkspaceEventImport"] = "importWorkspaceEvent";
    // Event category
    AdminPermission["AP_EventCategoryList"] = "listEventCategory";
    AdminPermission["AP_EventCategoryCreate"] = "createEventCategory";
    AdminPermission["AP_EventCategoryUpdate"] = "updateEventCategory";
    AdminPermission["AP_EventCategoryDelete"] = "deleteEventCategory";
    // Software plan
    AdminPermission["AP_ClientPlanList"] = "listClientPlan";
    AdminPermission["AP_ClientPlanCreate"] = "createClientPlan";
    AdminPermission["AP_ClientPlanUpdate"] = "updateClientPlan";
    AdminPermission["AP_ClientPlanDelete"] = "deleteClientPlan";
    // Service order
    AdminPermission["AP_ClientOrderView"] = "viewClientOrder";
    AdminPermission["AP_ClientOrderList"] = "listClientOrder";
    AdminPermission["AP_ClientOrderCreate"] = "createClientOrder";
    AdminPermission["AP_ClientOrderUpdate"] = "updateClientOrder";
    AdminPermission["AP_ClientOrderConfirm"] = "confirmClientOrder";
    AdminPermission["AP_ClientOrderDelete"] = "deleteClientOrder";
    // Transaction
    AdminPermission["AP_ClientTransactionList"] = "listClientTransaction";
    AdminPermission["AP_ClientTransactionCreate"] = "createClientTransaction";
    AdminPermission["AP_ClientTransactionUpdate"] = "updateClientTransaction";
    AdminPermission["AP_TransactionExport"] = "exportTransaction";
    // Unknown transaction
    AdminPermission["AP_UnknownTransactionList"] = "listUnknownTransaction";
    AdminPermission["AP_UnknownTransactionHandle"] = "handleUnknownTransaction";
    // Widget type
    AdminPermission["AP_WidgetTypeList"] = "listWidgetType";
    AdminPermission["AP_WidgetTypeCreate"] = "createWidgetType";
    AdminPermission["AP_WidgetTypeUpdate"] = "updateWidgetType";
    AdminPermission["AP_WidgetTypeDelete"] = "deleteWidgetType";
    AdminPermission["AP_WidgetDuplicate"] = "duplicateWidget";
    // Administrators
    AdminPermission["AP_AdminUserList"] = "listAdminUser";
    AdminPermission["AP_AdminUserUpdate"] = "updateAdminUser";
    AdminPermission["AP_AdminUserCreate"] = "createAdminUser";
    AdminPermission["AP_AdminUserDelete"] = "deleteAdminUser";
    AdminPermission["AP_AdminUserUpdatePassword"] = "updateAdminUserPassword";
    // Role
    AdminPermission["AP_AdminUserRoleList"] = "listRole";
    AdminPermission["AP_AdminUserRoleCreate"] = "createRole";
    AdminPermission["AP_AdminUserRoleUpdate"] = "updateRole";
    AdminPermission["AP_AdminUserRoleDelete"] = "deleteRole";
    // Users
    AdminPermission["AP_UserList"] = "listUser";
    AdminPermission["AP_UserLockRecover"] = "lockRecoverUser";
    AdminPermission["AP_UserUpdate"] = "updateUser";
    AdminPermission["AP_UserDelete"] = "deleteUser";
    AdminPermission["AP_UserCreate"] = "createUser";
    AdminPermission["AP_UserSignIn"] = "signInUser";
    // Event
    AdminPermission["AP_EventList"] = "listEvent";
    AdminPermission["AP_EventCreate"] = "createEvent";
    // Visitor
    AdminPermission["AP_VisitorList"] = "listVisitor";
    AdminPermission["AP_VisitorLockRecover"] = "lockRecoverVisitor";
    // End users
    AdminPermission["AP_EndUserList"] = "listEndUser";
    AdminPermission["AP_EndUserLockRecover"] = "lockRecoverEndUser";
    // Order report
    AdminPermission["AP_OrderList"] = "listOrder";
    // Site template
    AdminPermission["AP_SiteTemplateList"] = "listSiteTemplate";
    AdminPermission["AP_SiteTemplateCreate"] = "createSiteTemplate";
    AdminPermission["AP_SiteTemplateUpdate"] = "updateSiteTemplate";
    AdminPermission["AP_SiteTemplateDelete"] = "deleteSiteTemplate";
    // Page template
    AdminPermission["AP_PageTemplateList"] = "listPageTemplate";
    AdminPermission["AP_PageTemplateCreate"] = "createPageTemplate";
    AdminPermission["AP_PageTemplateUpdate"] = "updatePageTemplate";
    AdminPermission["AP_PageTemplateDelete"] = "deletePageTemplate";
})(AdminPermission || (AdminPermission = {}));
