// @breezingin/breezing-shared/dist/esm/enum/currency.js
export var Currency;
(function (Currency) {
    Currency["VND"] = "VND";
    Currency["USD"] = "USD";
    Currency["EUR"] = "EUR";
    Currency["GBP"] = "GBP";
    Currency["JPY"] = "JPY";
    Currency["KRW"] = "KRW";
    Currency["CNY"] = "CNY";
    Currency["SGD"] = "SGD";
    Currency["AUD"] = "AUD";
    Currency["CAD"] = "CAD";
    Currency["CHF"] = "CHF";
    Currency["HKD"] = "HKD";
    Currency["NZD"] = "NZD";
    Currency["THB"] = "THB";
    Currency["MYR"] = "MYR";
    Currency["IDR"] = "IDR";
    Currency["PHP"] = "PHP";
    Currency["INR"] = "INR";
    Currency["BRL"] = "BRL";
    Currency["RUB"] = "RUB";
    Currency["ZAR"] = "ZAR";
    Currency["MXN"] = "MXN";
    Currency["TRY"] = "TRY";
    Currency["SEK"] = "SEK";
    Currency["NOK"] = "NOK";
    Currency["DKK"] = "DKK";
    Currency["PLN"] = "PLN";
    Currency["CZK"] = "CZK";
    Currency["HUF"] = "HUF";
    Currency["ILS"] = "ILS";
    Currency["AED"] = "AED";
    Currency["SAR"] = "SAR";
    Currency["QAR"] = "QAR";
    Currency["KWD"] = "KWD";
    Currency["BHD"] = "BHD";
    Currency["OMR"] = "OMR";
    Currency["JOD"] = "JOD";
    Currency["EGP"] = "EGP";
    Currency["CLP"] = "CLP";
    Currency["COP"] = "COP";
    Currency["PEN"] = "PEN";
    Currency["ARS"] = "ARS";
    Currency["UYU"] = "UYU";
    Currency["VES"] = "VES";
    Currency["UAH"] = "UAH";
    Currency["RON"] = "RON";
    Currency["BGN"] = "BGN";
    Currency["HRK"] = "HRK";
    Currency["RSD"] = "RSD";
    Currency["ISK"] = "ISK";
    Currency["HNL"] = "HNL";
    Currency["GTQ"] = "GTQ";
    Currency["CRC"] = "CRC";
    Currency["PAB"] = "PAB";
    Currency["DOP"] = "DOP";
    Currency["TTD"] = "TTD";
    Currency["BBD"] = "BBD";
    Currency["JMD"] = "JMD";
    Currency["BSD"] = "BSD";
    Currency["KYD"] = "KYD";
    Currency["BMD"] = "BMD";
    Currency["FJD"] = "FJD";
    Currency["WST"] = "WST";
    Currency["TOP"] = "TOP";
    Currency["SBD"] = "SBD";
    Currency["VUV"] = "VUV";
    Currency["XPF"] = "XPF";
    Currency["CDF"] = "CDF";
    Currency["XAF"] = "XAF";
    Currency["XOF"] = "XOF";
    Currency["XCD"] = "XCD";
    Currency["ANG"] = "ANG";
    Currency["AWG"] = "AWG";
    Currency["BIF"] = "BIF";
    Currency["BWP"] = "BWP";
    Currency["CVE"] = "CVE";
    Currency["DJF"] = "DJF";
    Currency["ERN"] = "ERN";
    Currency["ETB"] = "ETB";
    Currency["GMD"] = "GMD";
    Currency["GNF"] = "GNF";
    Currency["KES"] = "KES";
    Currency["KMF"] = "KMF";
    Currency["LSL"] = "LSL";
    Currency["LRD"] = "LRD";
    Currency["LYD"] = "LYD";
    Currency["MAD"] = "MAD";
    Currency["MGA"] = "MGA";
    Currency["MRO"] = "MRO";
    Currency["MUR"] = "MUR";
    Currency["MWK"] = "MWK";
    Currency["MZN"] = "MZN";
    Currency["NAD"] = "NAD";
    Currency["NGN"] = "NGN";
    Currency["RWF"] = "RWF";
    Currency["SCR"] = "SCR";
    Currency["SDG"] = "SDG";
    Currency["SHP"] = "SHP";
    Currency["SLL"] = "SLL";
    Currency["SOS"] = "SOS";
    Currency["SZL"] = "SZL";
    Currency["TZS"] = "TZS";
    Currency["UGX"] = "UGX";
    Currency["ZMW"] = "ZMW";
    Currency["ZWL"] = "ZWL";
})(Currency || (Currency = {}));
