// @breezingin/breezing-shared/dist/cjs/schemas/event.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.EventInfoCollectionType = exports.EventCurrencyUnit = exports.EventLocationType = exports.EventRegisterMode = void 0;
var EventRegisterMode;
(function (EventRegisterMode) {
    EventRegisterMode["Open"] = "open";
    EventRegisterMode["ApprovalRequired"] = "approval-required";
    EventRegisterMode["InvitedOnly"] = "invited-only";
})(EventRegisterMode || (exports.EventRegisterMode = EventRegisterMode = {}));
var EventLocationType;
(function (EventLocationType) {
    EventLocationType["InPerson"] = "in-person";
    EventLocationType["Online"] = "online";
    EventLocationType["Hybrid"] = "hybrid";
})(EventLocationType || (exports.EventLocationType = EventLocationType = {}));
var EventCurrencyUnit;
(function (EventCurrencyUnit) {
    EventCurrencyUnit["USD"] = "USD";
    EventCurrencyUnit["VND"] = "VND";
})(EventCurrencyUnit || (exports.EventCurrencyUnit = EventCurrencyUnit = {}));
var EventInfoCollectionType;
(function (EventInfoCollectionType) {
    EventInfoCollectionType["PerTicket"] = "per-ticket";
    EventInfoCollectionType["PerOrder"] = "per-order";
})(EventInfoCollectionType || (exports.EventInfoCollectionType = EventInfoCollectionType = {}));
