// @breezingin/breezing-shared/dist/cjs/schemas/promotion.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.RequiredItemsMode = exports.PromotionApplyMethod = exports.PromotionType = void 0;
var PromotionType;
(function (PromotionType) {
    PromotionType["TicketDiscount"] = "ticket-discount";
    PromotionType["ProductDiscount"] = "product-discount";
    PromotionType["OrderDiscount"] = "order-discount";
    PromotionType["BuyXGetY"] = "buy-x-get-y";
    PromotionType["OrderGift"] = "order-gift";
    PromotionType["FreeShipping"] = "free-shipping";
})(PromotionType || (exports.PromotionType = PromotionType = {}));
var PromotionApplyMethod;
(function (PromotionApplyMethod) {
    PromotionApplyMethod["Auto"] = "auto";
    PromotionApplyMethod["Code"] = "code";
})(PromotionApplyMethod || (exports.PromotionApplyMethod = PromotionApplyMethod = {}));
var RequiredItemsMode;
(function (RequiredItemsMode) {
    RequiredItemsMode["All"] = "all";
    RequiredItemsMode["Any"] = "any";
    RequiredItemsMode["Count"] = "count";
})(RequiredItemsMode || (exports.RequiredItemsMode = RequiredItemsMode = {}));
