// @breezingin/breezing-shared/dist/cjs/schemas/user-role.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.EventAccessType = exports.UserPermission = void 0;
var UserPermission;
(function (UserPermission) {
    UserPermission["ViewDashboard"] = "view-dashboard";
    UserPermission["EventList"] = "event-list";
    UserPermission["EventCreate"] = "event-create";
    UserPermission["EventSetup"] = "event-setup";
    UserPermission["EventApproval"] = "event-approval";
    UserPermission["EventReport"] = "event-report";
    UserPermission["CheckInList"] = "check-in-list";
    UserPermission["CheckInCreate"] = "check-in-create";
    UserPermission["EventCustomFieldSetup"] = "event-custom-field-setup";
    UserPermission["SessionSetup"] = "session-setup";
    UserPermission["TicketCategorySetup"] = "ticket-category-setup";
    UserPermission["SeatMapSetup"] = "seat-map-setup";
    UserPermission["ReferralCodeList"] = "referral-code-list";
    UserPermission["ReferralCodeManage"] = "referral-code-manage";
    UserPermission["EventEmailCampaignList"] = "event-email-campaign-list";
    UserPermission["EventEmailCampaignCreate"] = "event-email-campaign-create";
    UserPermission["CouponCodeList"] = "coupon-code-list";
    UserPermission["CouponCodeManage"] = "coupon-code-manage";
    UserPermission["TicketList"] = "ticket-list";
    UserPermission["TicketUpdate"] = "ticket-update";
    UserPermission["TicketCreate"] = "ticket-create";
    UserPermission["OrderList"] = "order-list";
    UserPermission["OrderCreate"] = "order-create";
    UserPermission["WorkspaceSetup"] = "workspace-setup";
    UserPermission["UserManagement"] = "user-management";
    UserPermission["RoleManagement"] = "role-management";
    UserPermission["VendorList"] = "vendor-list";
    UserPermission["VendorManagement"] = "vendor-management";
    UserPermission["CreditTransactionList"] = "credit-transaction-list";
    UserPermission["CreditTransactionManagement"] = "credit-transaction-management";
    // Vendor permissions
    UserPermission["VendorEventList"] = "vendor-event-list";
    UserPermission["VendorCreditTransactionList"] = "vendor-credit-transaction-list";
    UserPermission["VendorCreditTransactionManagement"] = "vendor-credit-transaction-management";
})(UserPermission || (exports.UserPermission = UserPermission = {}));
var EventAccessType;
(function (EventAccessType) {
    EventAccessType["All"] = "all";
    EventAccessType["Scoped"] = "scoped";
    EventAccessType["Self"] = "self";
})(EventAccessType || (exports.EventAccessType = EventAccessType = {}));
