// @breezingin/breezing-shared/dist/cjs/schemas/widget-type.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.WidgetPropInput = exports.WidgetPropCategory = exports.WidgetTypeCategory = void 0;
var WidgetTypeCategory;
(function (WidgetTypeCategory) {
    WidgetTypeCategory["Default"] = "default";
    WidgetTypeCategory["Header"] = "header";
    WidgetTypeCategory["Footer"] = "footer";
    WidgetTypeCategory["Hero"] = "hero";
    WidgetTypeCategory["Form"] = "form";
    WidgetTypeCategory["CallToAction"] = "callToAction";
    WidgetTypeCategory["Agenda"] = "agenda";
    WidgetTypeCategory["Testimonial"] = "testimonial";
    WidgetTypeCategory["Gallery"] = "gallery";
    WidgetTypeCategory["General"] = "general";
    WidgetTypeCategory["Location"] = "location";
})(WidgetTypeCategory || (exports.WidgetTypeCategory = WidgetTypeCategory = {}));
var WidgetPropCategory;
(function (WidgetPropCategory) {
    WidgetPropCategory["Content"] = "content";
    WidgetPropCategory["Layout"] = "layout";
    WidgetPropCategory["Style"] = "style";
    WidgetPropCategory["Advance"] = "advance";
    WidgetPropCategory["Setting"] = "setting";
})(WidgetPropCategory || (exports.WidgetPropCategory = WidgetPropCategory = {}));
var WidgetPropInput;
(function (WidgetPropInput) {
    WidgetPropInput["Text"] = "text";
    WidgetPropInput["Textarea"] = "textarea";
    WidgetPropInput["Number"] = "number";
    WidgetPropInput["Image"] = "image";
    WidgetPropInput["Select"] = "select";
    WidgetPropInput["Switch"] = "boolean";
    WidgetPropInput["CodeBlock"] = "code-block";
    WidgetPropInput["Object"] = "object";
    WidgetPropInput["Array"] = "array";
    WidgetPropInput["Color"] = "color";
    WidgetPropInput["TicketRegistrationForm"] = "ticket-registration-form";
    WidgetPropInput["TicketRSVPForm"] = "ticket-rsvp-form";
    WidgetPropInput["CheckoutForm"] = "checkout-form";
})(WidgetPropInput || (exports.WidgetPropInput = WidgetPropInput = {}));
