// @breezingin/breezing-shared/dist/cjs/enum/seat-map.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.SeatObjectType = exports.SelectionType = exports.NodeType = void 0;
var NodeType;
(function (NodeType) {
    NodeType["RECTANGLE"] = "Rectangle";
    NodeType["CIRCLE"] = "Circle";
    NodeType["GROUP"] = "Group";
    NodeType["TEXT"] = "Text";
    NodeType["TRANSFORMER"] = "Transformer";
    NodeType["SHAPE"] = "Shape";
})(NodeType || (exports.NodeType = NodeType = {}));
var SelectionType;
(function (SelectionType) {
    SelectionType["SEAT"] = "seat";
    SelectionType["OBJECT"] = "object";
    SelectionType["COMBINATION"] = "combination";
    SelectionType["SEAT_ROW"] = "seat-row";
    SelectionType["SEAT_TABLE"] = "seat-table";
    SelectionType["DEFAULT"] = "default";
    SelectionType["TEXT"] = "text";
})(SelectionType || (exports.SelectionType = SelectionType = {}));
var SeatObjectType;
(function (SeatObjectType) {
    SeatObjectType["SEAT_TABLE"] = "g-obj-table";
    SeatObjectType["SEAT_ROW"] = "g-obj";
    SeatObjectType["SEAT"] = "c-obj";
    SeatObjectType["TEXT"] = "obj-text";
    SeatObjectType["OBJECT"] = "obj";
})(SeatObjectType || (exports.SeatObjectType = SeatObjectType = {}));
